manual/add_ovf_properties.sh

```diff
@@ -3,25 +3,33 @@
 # SPDX-License-Identifier: BSD-2
 
 OUTPUT_PATH="../output-vmware-iso"
+OVF_PATH=`find ${OUTPUT_PATH} -type f -iname ${VEBA_APPLIANCE_NAME}.ovf -exec dirname "{}" \;`
```

Comment:
Nit: Could we use `$()` notation foe ease of reading to encapsulate the command rather than ```?

Reply:
Yep! Have fixed that and updated.